js/app.js

```diff
@@ -0,0 +1,65 @@
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const books = document.querySelector('#books-list');
+const addBtn = document.getElementById('form');
+const rmvBtn = document.getElementById('remove-btn');
+let booksArray;
+
+function displayBooks() {
+  const data = localStorage.booksList;
+  if (data === null) {
+    booksArray = [];
+  } else {
+    booksArray = JSON.parse(data);
+  }
+
+  let htmlCode = '';
+  booksArray.forEach((item, index) => {
+    htmlCode += `<div>
+    <h2>${item.title}</h2>
+    <p>${item.author}</p>
+    <button onclick='removeBook(${index})' id='remove-btn'>remove</button>
+    <hr>
+    </div>`;
+  });
+  books.innerHTML = htmlCode;
+}
+
+displayBooks();
+
+function addBook(title, author) {
+  const data = localStorage.booksList;
+  if (data === null) {
+    booksArray = [];
+  } else {
+    booksArray = JSON.parse(data);
+  }
+  booksArray.push({
+    title,
+    author,
+  });
+
+  localStorage.booksList = JSON.stringify(booksArray);
+  displayBooks();
+}
```

Comment:
- [ ] good job setting up the project 👍 however when adding a book nothing is added to the book array because your page is reloading.
- [ ] Also nothing is added to localStorage because there is no `setItem` or `getItem` used properly